code/modules/mob/living/carbon/metroid/powers.dm

```diff
@@ -1,74 +1,79 @@
 /mob/living/carbon/slime/proc/Wrap(var/mob/living/M) // This is a proc for the clicks
-	if (Victim == M || src == M)
+	if(victim == M || src == M)
 		Feedstop()
 		return
 
-	if (Victim)
-		to_chat(src, "I am already feeding...")
+	if(victim)
+		to_chat(src, span("warning", "I am already feeding..."))
 		return
 
-	var t = invalidFeedTarget(M)
-	if (t)
-		to_chat(src, t)
+	var/target = invalidFeedTarget(M)
+	if(target)
+		to_chat(src, target)
 		return
 
 	Feedon(M)
 
 /mob/living/carbon/slime/proc/invalidFeedTarget(var/mob/living/M)
-	if (!M || !istype(M))
-		return "This subject is incomparable..."
-	if (istype(M, /mob/living/carbon/slime)) // No cannibalism... yet
+	if(!M || !istype(M))
+		return "This subject is incompatible.."
+	if(istype(M, /mob/living/carbon/slime)) // No cannibalism... yet
 		return "I cannot feed on other slimes..."
-	if (!Adjacent(M))
+	if(!Adjacent(M))
 		return "This subject is too far away..."
-	if (istype(M, /mob/living/carbon) && M.getCloneLoss() >= M.maxHealth * 1.5 || istype(M, /mob/living/simple_animal) && M.stat == DEAD)
+	if(istype(M, /mob/living/carbon) && M.getCloneLoss() >= M.maxHealth * 2 || istype(M, /mob/living/simple_animal) && M.stat == DEAD)
```

Comment:
This is a LOT of health. `maxHealth * 2` is 400 for humans. Are you sure?

Reply:
Yeah. In my playtesting, I found maxHealth * 2 to be a really comfortable number, especially with monkeys. It allows the adult slime to suckle a bit more on the thing, which causes it to split more nicely. Otherwise you have "fractions" of monkeys used to fuel a single evolution + reproduction combo. If this turns out to be REALLY bad, another check can be added in the future.